1_fetch/src/get_sb.R

```diff
@@ -1,11 +1,11 @@
 #' download file from Sciencebase
 #' @param download_file char file to download
 #' @param destination_file char path and name to save downloaded file as
-download_sciencebase <- function(download_file, destination_file) {
+download_sciencebase <- function(out_filepath) {
 
   item_file_download('5d925066e4b0c4f70d0d0599', 
-                     names = download_file, 
-                     destinations = destination_file, 
+                     names = 'me_RMSE.csv', 
+                     destinations = out_filepath, 
                      overwrite_file = TRUE)
```

Comment:
Since `download_sciencebase` is being called to build a file target, we'll need to `return(out_filepath)` at the end of this function

Reply:
Unless maybe `item_file_download()` returns the filepath automatically, which it may if this didn't error when you ran it. And I see you did add that return call elsewhere, so am guessing that's the case?

Reply:
Yes, it does return the file path on its own